src/cli/data.py

```diff
@@ -260,17 +209,18 @@ def list(self):
     def describe(self):
         """describe file or row or database"""
 
-        if PREFIXES.FILE in self.app.pargs.object_id:
-            key_label = "Property"
+        key_label = "Property"
```

Comment:
naming the first column "Column" was very confusing, especially because i now make a new table of columns